cutting-a-release.md

```diff
@@ -1,47 +1,81 @@
-- [ ] Most repos have a section in either the README or the Contributing section of the docs that describes how to use the `make release` script. It looks something like [this](https://web3py.readthedocs.io/en/stable/contributing.html#releasing). Follow those instructions. Usually, it looks something like `$ make release bump=$$version part to bump$$`.
-  - Troubleshooting:
-    - If something goes wrong in the middle, you'll need to figure out what went wrong and fix that problem
-    - Open up the Makefile, and run the failed command and all subsequent commands manually
-    - If the problem persists, you may be able to skip the command and run the subsequent commands manually
-- [ ] Test it!
-  - Create and activate a new virtualenv
-  - `$ pip install -U <released_project>`
-  - Open shell and confirm that the module import works. Check a simple feature and/or new features
-- [ ] Make sure the ReadTheDocs build succeeded. Visit the `https://<project name>.readthedocs.io/en/stable/releases.html` and make sure your changes are there.
-- [ ] Announce to the Discord channel if it's a user-facing library. Typically this announcement includes a link to the release notes. :tada:
+Releases should be considered for each change that is merged to the main branch. Not all merges require a new version so it is reasonable to bundle several smaller changes together in a single release. All libraries should follow Continuous Delivery and Semver best practices. We typically release on Wednesdays and at minimum once per month (but could be longer for less prominent libraries).
+
+> :warning: Make sure to cut the release when there is time to support it if something goes wrong. As a rule, don't do releases late in the afternoon or on Fridays.
+
+The team should review any outstanding PRs and recent merges at the weekly sync. If there are any newsfragments in the main branch there is a good chance a release is needed.
+
+How to cut a new release
+====
+
+A release of a library requires a version bump, compiling release notes, and publishing to PyPI.
+
+### Releasing with Semver
+
+We use [Semantic Versioning](https://semver.org/) for all of our libraries. Decide which bump applies (major, minor, or patch) to the changes that will be relased. Here's a quick cheat sheet:
+  - Use a `major` version bump when you make incompatible API changes
+  - Use a `minor` version bump when you add functionality in a backwards-compatible manner
+  - Use a `patch` version bump when you make backwards-compatible bug fixes
+
+Follow the sections below to release with the desired version.
+
+### Requirements
+
+  1. Changes are merged in the main branch
+  2. The latest build on the main branch was successful
+
+### Steps
+
+  1. From your local copy of the lib's repository, check out and pull the latest from the main branch.
+  1. Activate your virtual environment and run `python -m pip install -e ".[dev]"` to ensure the latest dev dependencies are installed.
```

Comment:
These bullets are all numbered `1`. I think it still renders fine in rich form but would be good to correct it in case the rich form isn't being viewed.

Reply:
Sounds good, the list isn't terribly long and wouldn't take much to add something later.